src/test/datascience/notebook/contentProvider.vscode.test.ts

```diff
@@ -201,13 +201,21 @@ suite('DataScience - VSCode Notebook - (Open)', function () {
 
         const originalJsonStr = (await fs.readFile(templateIPynb, { encoding: 'utf8' })).trim();
         const originalJson: nbformat.INotebookContent = JSON.parse(originalJsonStr);
-        assert.deepEqual(JSON.parse(model.getContent()), originalJson, 'Untrusted notebook json content is invalid');
+        assert.deepEqual(
+            JSON.parse(model.getContent()).cells,
+            originalJson.cells,
```

Comment:
This was failing because we change the kernel in the metadata. That content is not worth comparing, so lets just compare the cells.

Reply:
This is not correct, if the user didn't make any changes but just trusted the notebook, then we should not be making changes. 
I.e. just because a notebook is trusted doesn't mean we should modify the file.
I think its a bug.

Reply:
When would it fair to change a notebook's metadata? I get not when opening or trusting. How about when a kernel started? If so, that's our default behavior.

Reply:
> w about when a kernel started? If so, that's our default behavior.

Yes that's find, but thiis test doesn't do anything like that, this is just trusting a notebook, hence metadata should not change.

Reply:
I think this is ok David. 
Let's fix this test, i can file a separate issue for what I'm describing

Reply:
Here's the new issue: https://github.com/microsoft/vscode-jupyter/issues/5721